src/config/tomlconfig.rs

```diff
@@ -11,6 +11,8 @@ pub struct TomlConfig {
     pub description: Option<String>,
 
     pub output: Option<TomlOutputConfig>,
+
+    pub use_ace: Option<bool>,
```

Comment:
Hi if I might suggest something. Maybe we could we have ACE as an implementation detail and change the toml/confg entry to something along the lines of:
`playpen-editable: Option<bool>` or even
`playpen-properties:Option<Vec<PlaypenProperty>>` where editability / resettability would be just one of the properties?

For instance some books might not want to enable clipboard support or play button  too.